src/lib.rs

```diff
@@ -481,6 +481,43 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
         }
     }
 
+    ///Returns contents of Vob as Vec of bytes.
+    ///
+    ///For a Vob of length n, the first bit will be the 0th element,
+    ///the last bit would be the (n-1)th element.
+    ///
+    /// # Examples
+    /// ```
+    ///use vob::Vob;
+    ///let mut x = Vob::from_elem(5, false);
+    ///x.set(2, true);
+    ///x.set(4, true);
+    ///assert_eq!(x.to_bytes(), [0b00101000]);
+    ///```
+    pub fn to_bytes(&self) -> Vec<u8> {
+        fn bit<T: Debug + One + PrimInt + Zero>(vob: &Vob<T>, byte: usize, bit: usize) -> u8 {
+            let offset = byte * 8 + bit;
+            if offset >= vob.len {
+                0
+            } else {
+                (vob[offset] as u8) << (7 - bit)
+            }
+        }
```

Comment:
This seems unnecessary, as most of this already happens in `get` (except for the shift).